code/game/jobs/job/offduty_vr.dm

```diff
@@ -98,6 +98,7 @@
 	pto_type = PTO_SECURITY
 	economic_modifier = 4
 
+/*
```

Comment:
This should be kept, so that people who have exploration off-duty hours stored up can still use them.